src/shared/seccomp-util.c

```diff
@@ -944,25 +960,35 @@ int seccomp_add_syscall_filter_item(scmp_filter_ctx *seccomp, const char *name,
                                 return r;
                 }
 
+                if (ret_added) {
+                        r = strv_extend(ret_added, name);
```

Comment:
not a fan of naming parameters that are not output-only `ret_`, because that suggests we'd only overwrite the parameter, never read it. Hence drop the `ret_` prefix here...

(that said, I think the functioon exported to other code ideally would have an output-only param. i.e. ideally only the internal helper functions use params for this that are both input and output)

Reply:
I renamed the parameter, but I didn't make change the interface otherwise. This function has essentially one user, and making the parameter output-only would complicate that user, and slow things down (because we'd need to concat the strvs).